### DISCUSSION TITLE
求助！崩铁支配恶兽无法6600分！

### DISCUSSION THREAD

**Discussion by hyz2333333** [share121/inter-knot#29] [其他]
![1722227218639](https://github.com/user-attachments/assets/63cb0a09-86e3-4f90-a828-899bd84e96e4)
![S40809-11051509_com miHoYo hkrpg bilibili](https://github.com/user-attachments/assets/761a91b7-d5f9-435e-8bcb-b219b507300e)
我的真理2＋2萨尔所图的60/190带符玄和如泥上半没伤害！
卡黑的面板不高而且这期没有任何环境！
超击破队不能拆开成两半用！
没有龙丹，没有托帕，没有云璃！
雀神这模式感觉也没伤害，雪衣破弱点后输出不佳！
这期黄泉被策划恶意针对了！上半雷抗！我原本可以下半走超击破！求绳网的人支个招
而且这期生存压力也很大！

**Comment by BiaoBiaoB** [comment]
嗯呐！嗯呐！嗯呐哒！（这个时候就到我出场了！你怎么知道我垫池子垫出了双金！现在一命云璃了！）

**Comment by ghost** [comment]
楼上扎我心了。

**Comment by koyki233** [comment]
遇事不决，量子破船

**Comment by morning51** [comment]
我第四关直接没打

**Comment by Mu-7593** [comment]
这次硬打不好打了，我上半和楼主阵容一样无压力过，下半只能老老实实用克拉拉+停云+藿藿+知更鸟；带追击Buff，凹了七次才勉强过的
